Dream-Home-PEABOT/FEE_DreamHome
# Context adjustment

### What does this PR do?

* Adds dynamic rendering for the question being displayed to the user
* Adds logic to survey component for which question to render
* Adds local state to question
* Adds ability for question to alter survey state and answerContext

### Where should the reviewer start?

* Start by checking naming convention for altered files, ensure consistency

### How should this be manually tested?

* Get all prerequisite helpers running
* Input info into input field, press button and use dev tools to check state change

### Any background context you want to provide?

* It seems we will be able to conditionally render the question component pretty seamlessly with this logic
* We will need to implement some further logic to show other relevant question info (only question name currently displaying)

### What are the relevant tickets?

closes #38 
closes #39 
closes #40 
closes #41 

### Future notes:

* Add history capabilities to state so a user can move back questions
* Add logic to display current questions other relevant info

### Screenshots (if appropriate)

 <img src="https://media0.giphy.com/media/WdtWhAodICNGuUlUgK/giphy.gif" alt="" height=100% width=80%/>
